README.md

```diff
@@ -0,0 +1,104 @@
+# SignalFx Go Lambda Wrapper
+
+SignalFx Golang Lambda Wrapper.
+
+## Usage
+
+The SignalFx Go Lambda Wrapper is a wrapper around an AWS Lambda Go function handler, used to instrument execution of the function and send metrics to SignalFx.
+
+### Install via maven dependency
+To install 
+
+`$ go get https://github.com/signalfx/lambda-go`
+
+### Environment Variable
+Set the required environment variable below:
+
+`SIGNALFX_AUTH_TOKEN=<signalfx token>`
+
+Optionally the environment variables below can be set:
+
+`SIGNALFX_INGEST_ENDPOINT=<ingest endpoint, default=https://ingest.signalfx.com/v2/datapoint>`
```

Comment:
Just say `SIGNALFX_INGEST_ENDPOINT=https://ingest.signalfx.com/v2/datapoint` and make a note above that the values shown are the default.